index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+
+    <div class="container">
+        <div class="header-nav">
+            <p class="MyLogo">My Logo</p>
+            <div class="topNav">
```

Comment:
[ ] you are working on the mobile version of your portfolio, the navbar should contain a hamburger button instead of a list of elements, kindly add a hamburger button to your navbar .👀
Figma design :
 
![figmalog](https://user-images.githubusercontent.com/62156607/193100963-a3f6ccf8-f4b2-49da-aff2-85788c3f1564.PNG)

your design : 

![hisnavbar](https://user-images.githubusercontent.com/62156607/193101179-a38f2246-d535-43b3-80e0-95379366019a.PNG)